packages/cacti-copm-core/src/main/typescript/lib/types.ts

```diff
@@ -1,7 +1,8 @@
 export type DLTransactionParams = {
   contract: string;
```

Comment:
Sorry, I didn't notice this earlier. Can you rename this to `contractId`? That's just more meaningful.

Reply:
This has been renamed.